### PR TITLE
DOCS-1023 and DOCS-1027: Add control views to motor and encoder

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: static/include/components/test-control/motor-control.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Test the motor
+
+Once your motor is configured and connected, go to the **Control** tab and click on the motor's drop-down panel.
+Use the buttons to try turning your motor forwards or backwards at different power levels and check whether it is moving as expected.
```
Comment (JessamyT) [inline static/include/components/test-control/motor-control.md:4]:
```suggestion
Use the buttons to try turning your motor forwards or backwards at different power levels and check whether it moves as expected.
```

File: static/include/components/test-control/encoder-control.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Test the encoder
+
+Once your encoder is configured and connected, go to the **Control** tab and click on the encoder's drop-down panel.
```
Comment (npentrel) [inline static/include/components/test-control/encoder-control.md:3]:
```suggestion
Once your encoder is configured and connected, go to the [**Control** tab](/manage/fleet/robots/#control) and click on the encoder's drop-down panel.
```

File: static/include/components/test-control/motor-control.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## Test the motor
+
+Once your motor is configured and connected, go to the **Control** tab and click on the motor's drop-down panel.
```
Comment (npentrel) [inline static/include/components/test-control/motor-control.md:3]:
```suggestion
Once your motor is configured and connected, go to the [**Control** tab](/manage/fleet/robots/#control) and click on the motor's drop-down panel.
```

**Review by npentrel — COMMENTED** [review]
Can you add them to the fake models too?

**Comment by JessamyT** [comment]
@npentrel I can, but it wouldn't really be "testing" for fake components since there's nothing to test--do you still think it makes sense to add them there (maybe with custom text instead of the snippet)?

**Comment by npentrel** [comment]
Yeah I think it's more about ensuring people know they can go to the control tab so I think it makes sense to do it everywhere

**Review by npentrel — APPROVED** [review]

**Pull request merged**